src/object/srv_obj_migrate.c

```diff
@@ -394,9 +395,11 @@ struct migrate_pool_tls_create_arg {
 	uuid_t  co_hdl_uuid;
 	d_rank_list_t *svc_list;
 	uint64_t max_eph;
-	uint64_t global_version;
+	uint32_t global_version;
+	uint32_t layout_version;
 	int	version;
 	uint32_t opc;
+	uint32_t new_layout_ver;
```

Comment:
same, put the 3 version together and explain a little bit is helpful, thx

Reply:
sure